data-model/publication-flow.md

```diff
@@ -43,6 +43,11 @@ Iedere publication flow omvat volgende, vaste procedurestappen:
 1. Vertaling
 2. Publicatie
 
+((TBC Met de aanpassing van designs krijgen we:
```

Comment:
De definitie van wat procedurestappen zijn binnen een publicatieaangelegenheid in het data model staat los van hoe ze weergegeven worden in de GUI. Als er morgen beslist wordt om de verbeteringen aan vertalingen ook als aparte tab weer te geven, wil dat daarom niet zeggen dat het data model wijzigt. Vertaling blijft een procedurestap.

Zo lijkt het mij ook te zijn voor publicatie. Na het vertalen volgt een procedurestap om het besluit gepubliceerd te krijgen in het BS. Daarvoor zijn verschillende activiteiten nodig: aanvraag drukproef, verbetering drukproef, publicatie, enz. Maar dat zijn allemaal activiteiten binnen dezelfde eenheid van werk "het besluit publiceren in het BS"